modules/mongodb/manifests/s3backup/backup.pp

```diff
@@ -153,7 +153,10 @@
     owner   => $user,
     group   => $user,
     mode    => '0755',
-    require => User[$user],
+    require => [
+      User[$user],
```

Comment:
this require isn't required as the user is explicitly required when specifying above